src/components/UI/Text/InputNumber.vue

```diff
@@ -257,7 +258,7 @@ export default {
     },
     keyPress(event) {
       // invoked when a key is pressed
-      if (this.maxLength != null && this.localValue.length == this.maxLength) {
+      if (this.maxLength != null && this.localValue.toString().length == this.maxLength) {
```

Comment:
wouldn't it be better to change the localValue to string where it is computed in the first place instead of converting it to string every time?

Reply:
yes done

Reply:
returning `localvalue` as a string from get method